metrics/metrics.go

```diff
@@ -261,6 +273,7 @@ func bindMetrics() error {
 		reshareState,
 		reshareLeader,
 		OutgoingConnectionState,
+		IsDrandNode,
```

Comment:
the new metrics need to be registered here

Reply:
D'oh. Same thing that happened when I originally added `IsDrandNode`. Fixed and thanks!